book_maker/cli.py

```
@@ -13,7 +13,60 @@ def parse_prompt_arg(prompt_arg):
+            if "user" not in prompt or not prompt["user"]:
+                raise ValueError(
+                    "PromptDown file must contain at least one user message"
+                )
+
+            print(f"Successfully loaded PromptDown file: {prompt_arg}")
+
+            # Validate required placeholders
+            if any(c not in prompt["user"] for c in ["{text}"]):
+                raise ValueError(
+                    "User message in PromptDown must contain `{text}` placeholder"
+                )
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)